config/secrets.yml

```diff
@@ -212,18 +234,24 @@ test:
         secure: false
         httponly: true
 
-  cms_url: https://cms-dev.openstax.org
+  cms_url: https://dev.openstax.org
   openstax_url: https://dev.openstax.org
 
-  cms_api_url: https://cms-dev.openstax.org/apps/cms/api/
+  cms_api_url: https://dev.openstax.org/apps/cms/api/
   sentry:
       dsn: <%= ENV['SENTRY_DSN'] || 'invalid' %>
 
+  rescue_from:
+    raise_exceptions: <%= ActiveModel::Type::Boolean.new.cast ENV.fetch('RESCUE_FROM_RAISE_EXCEPTIONS', false) %>
+    raise_background_exceptions: <%= ActiveModel::Type::Boolean.new.cast ENV.fetch('RESCUE_FROM_RAISE_BACKGROUND_EXCEPTIONS', false) %>
+
 production:
```

Comment:
is this used in production @Dantemss ? (vs pulling from parameter store)

Reply:
even if not, won't the rescue from config complain if they are not set?

Reply:
Yeah for tutor-deployment and bit-deployment I decided to keep the production secrets template in this file as is standard and add those ENV variable substitutions for the production secrets. The deployment script writes a .env file from the parameter store secrets instead, that way I can do proper type casting in secrets.yml and the deployment itself is not ruby-specific.

Reply:
@jpslav please unresolve this comment if this doesn't address your concern